framework/test/jest/integration/utils/chain/transaction.js

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+
+'use strict';
+
+const { getKeys } = require('@liskhq/lisk-cryptography');
+const {
+	TransferTransaction,
+	SecondSignatureTransaction,
+	DelegateTransaction,
+	VoteTransaction,
+	MultisignatureTransaction,
+} = require('@liskhq/lisk-transactions');
+
+const createTransferTransaction = (passphrase, recipientId, amount) => {
+	const transaction = new TransferTransaction({
+		recipientId,
+		amount,
+	});
+	transaction.sign(passphrase);
+	return transaction.toJSON();
+};
+
+const createSecondSignatureTransaction = (passphrase, secondPassphrase) => {
+	const { publicKey } = getKeys(secondPassphrase);
+	const transaction = new SecondSignatureTransaction({
+		asset: {
+			signature: {
+				publicKey,
+			},
+		},
+	});
+	transaction.sign(passphrase);
+	return transaction.toJSON();
+};
+
+const createDelegateTransaction = (passphrase, username) => {
+	const transaction = new DelegateTransaction({
+		asset: {
+			delegate: {
+				username,
+			},
+		},
+	});
+	transaction.sign(passphrase);
+	return transaction.toJSON();
+};
+
+const createVoteTransaction = (passphrase, upvotes = [], downvotes = []) => {
+	const signedUpvotes = upvotes.map(v => `+${v}`);
+	const signedDownvotes = downvotes.map(v => `-${v}`);
+	const transaction = new VoteTransaction({
+		asset: {
+			votes: [...signedUpvotes, ...signedDownvotes],
+		},
+	});
+	transaction.sign(passphrase);
+	return transaction.toJSON();
+};
+
+const createMultiSignatureTransaction = (
+	passphrase,
+	min,
+	lifetime,
+	keysgroup,
+) => {
+	const signedPublicKeys = keysgroup.map(v => `+${v}`);
+	const transaction = new MultisignatureTransaction({
+		asset: {
+			multisignature: {
+				min,
+				lifetime,
+				keysgroup: signedPublicKeys,
+			},
+		},
+	});
+	transaction.sign(passphrase);
+	return transaction.toJSON();
+};
+
+const getTransaction = async (storage, id) =>
+	storage.entities.Transaction.getOne({ id }, { extended: true });
```

Comment:
Similar to previous comment, I don't why we need such 1 line wrappers.  If we have to pass `storage` and `id` then why can't we have that 1 line in the test itself.

Reply:
The purpose is to wrap the storage usage as much as possible, so that even if it's used in everywhere, we can change the test easily

Reply:
In that chase, I would suggest to group them for `storageUtils` not the `chainUtils`.

Reply:
I think getting the transaction is using storage, but not really responsibility of `storage`.
How storage is used is defined in each utils

Reply:
If you wrap this code `storage.entities.Transaction.get()`, its a storage util function, nothing else.

Reply:
I think it make more sense to bundle by domain, it will be easier to find the code too.
It bit strange other utils for account are in one place, and other function related to account is in other place

Reply:
Yes I agree and domain is already well defined. So why we have all database entities together in once place. We didn't put `Block` entity to block module and `Transaction` entity to transaction module. 

Please follow same pattern here, and group all `storage` related functions to `storageUtils`